packages/starlight/utils/route-data.ts

```diff
@@ -66,9 +66,10 @@ function getToC({ entry, locale, headings }: PageProps) {
 	};
 }
 
-function getLastUpdated({ entry, id }: PageProps): Date | undefined {
+function getLastUpdated({ entry, id, isFallback }: PageProps): Date | undefined {
```

Comment:
And then this is also simpler!

```suggestion
function getLastUpdated({ entry }: PageProps): Date | undefined {
```